src/testFixtures/workspaceFolder/csharp2.1-plain-sam-app/src/HelloWorld/obj/Debug/netcoreapp2.1/HelloWorld.AssemblyInfo.cs

```diff
@@ -0,0 +1,23 @@
+//------------------------------------------------------------------------------
```

Comment:
This looks like a `.gitignore` mistake. `obj/` files are typically not committed to source control.

Reply:
Yeah, this came from juggling branches. Removing...